### PR TITLE
simplified code

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

**Comment by tanjunchen** [comment]
@sougou

File: go/vt/vttablet/tabletserver/schema/schema.go [hunk 1]
```diff
@@ -217,10 +217,7 @@ func (ta *Table) HasPrimary() bool {
 
 // IsTopic returns true if TopicInfo is not nil.
 func (ta *Table) IsTopic() bool {
-	if ta.TopicInfo == nil {
-		return false
-	}
-	return true
+	return ta.TopicInfo == nil
```
Comment (deepthi) [inline go/vt/vttablet/tabletserver/schema/schema.go]: that should be `ta.TopicInfo != nil`

Comment (tanjunchen) [reply go/vt/vttablet/tabletserver/schema/schema.go]: OK Thank You

**Comment by tanjunchen** [comment]
@deepthi

**Pull request merged**